### DISCUSSION TITLE
Firefly

### DISCUSSION THREAD

**Discussion by robumb** [htpcBeginner/docker-traefik#328] [Q&A]
Hi, i wanted to test out the firefly app but when i try starting the container i get
`env file /home/$USER/docker/appdata/firefly/importer.env not found: stat /home/$USER/docker/appdata/firefly/importer.env: no such file or directory`. What should that file contain and what permisions should it have?

**Comment by marrrc** [comment]
see the paragraph "Download configuration files" here: https://docs.firefly-iii.org/how-to/firefly-iii/installation/docker/
